README.md

```diff
@@ -2,127 +2,156 @@
 
 ## Description
 
-VTEX App that shows a collection of products.
+VTEX App that shows a shelf of products. This is a VTEX app that is used by Dreamstore product.
```

Comment:
It is according the outdated template, please, follow the newest one from [Minicart](https://github.com/vtex-apps/minicart/edit/master/README.md) .